tests/getters/ranges.test.ts

```diff
@@ -49,3 +50,12 @@ $end: -1;
 }
 `.trim())
 })
+
+//test decimal-range
+test("integration", () => {
+	expect(errSass(`decimal-range(0, 0.1)`)).toThrowError("0 0.01 0.02 0.025 0.03 0.04 0.05 0.06 0.07 0.075 0.08 0.09 0.1")
```

Comment:
This is great. I would just add some code to check that this works the same for negative values. Nice work!